Gopkg.toml

```diff
@@ -20,7 +20,7 @@ required = [
 
 [[override]]
   name = "github.com/knative/pkg"
-  branch = "master"
+  branch = "release-0.7"
```

Comment:
Why is this showing as a diff, it's in HEAD.

Reply:
Should all be gone after a rebase